docs/lab/results/critical-results.mdx

```diff
@@ -0,0 +1,89 @@
+---
+title: "Critical Results"
+---
+
+When our Lab partners return results to Vital, they may return specific interpretations for each Marker:
+
+- `normal`, the marker value is inside the lab reference range.
+- `abnormal`, the marker value is outside the lab reference range.
+- `critical`, the marker value falls in the critical range for that marker, which can be life-threatening.
+
+These values are returned in an `interpretation` field for results, both in each individual marker and the overall results object:
+
+```json Resuls Object
+{
+  "metadata": {
+    "age": 19,
+    "dob": "18/08/1993",
+    "clia_number": "12331231",
+    "patient": "Bob Smith",
+    "provider": "Dr. Jack Smith",
+    "laboratory": "LabCorp",
+    "date_reported": "2020-01-01",
+    "date_collected": "2022-02-02",
+    "specimen_number": "123131",
+    "date_received": "2022-01-01",
+    "status": "final",
+    "interpretation": "critical"
+  },
+  "results": [
+    {
+      "name": "Hemoglobin",
+      "slug": "hemoglobin",
+      "unit": "g/dL",
+      "notes": null,
+      "value": 100.2,
+      "timestamp": null,
+      "max_range_value": 17.7,
+      "min_range_value": 13.0,
+      "is_above_max_range": true,
+      "is_below_min_range": false,
+      "interpretation": "critical"
+    },
+    {
+      "name": "Sodium",
+      "slug": "unknown",
+      "unit": "mmol/L",
+      "notes": null,
+      "value": 136,
+      "timestamp": null,
+      "max_range_value": 144,
+      "min_range_value": 134,
+      "is_above_max_range": false,
+      "is_below_min_range": false,
+      "interpretation": "normal"
+    }
+  ]
+}
+```
+
+Besides the `interpretation` field, there is also the `status` field, which can assume the values of `final` or `partial`, as the critical results are notified as soon as they're available, not all tests are guaranteed to have been processed by the lab.
+
+For critical results, the Lab that published the results must call a physician to contact the patient to inform them about the critical results and give the correct orientation. The physician called will be the Vital assigned physician who acts as the ordering physician when placing the Lab Test order. After the call has been made and Vital has received the results from the Lab, a `labtest.result.critical` event will be fired to customers, with the webhook payload containing the following information:
```

Comment:
For critical results, the laboratory will contact a physician, so they can provide further information to the patient.  The physician called is the Vital assigned physician. They act as the ordering physician when placing the order. After the call is made, and Vital has received the results from the laboratory, a `labtest.result.critical` event is sent. The webhook payload contains the following information: